src/deserialize.js

```diff
@@ -36,6 +36,11 @@ function reviveRecord(key, recInfo, options) {
     revivedData = RecordType.migrate(revivedData)
   }
 
+  const deserializer = options.deserializers && options.deserializers[recInfo['__record']]
```

Comment:
It would be cleaner to default the `options.deserializers` to an empty object (`{}`) in `revive()`.

The documentation would then also say `deserializers={}`.